etc/requirements.txt

```diff
@@ -14,6 +14,12 @@ numpy==1.11.3
 # for pandas-datareader
 requests-file==1.4.1
 
+# for requests
+certifi==2018.8.24
```

Comment:
Do we need to fix our build to handle pinning this new requirement, like we saw with https://github.com/quantopian/zipline/pull/2331 ?